packages/apps/composer-app/src/playwright/collections.spec.ts

```diff
@@ -91,5 +91,31 @@ test.describe('Collection tests', () => {
       await host.toggleCollectionCollapsed(0);
       await expect(host.getObjectLinks()).toHaveCount(2);
     });
+
+    test('deletion undo restores collection', async () => {
+      await host.closeToast();
```

Comment:
this won't work in CI, need to just figure out why you're needing to do this locally? what's your `DX_ENVIRONMENT` set to?

Reply:
Hmm, for some reason my `DX_ENVIRONMENT` is my computer name.

Reply:
yup, that's what mine is as well. Does it not end in `.local` or `.lan` mine automatically does, hence the notify condition in the observability plugin being based on that

Reply:
It's `zan.mbp`. I'll add an export!